alembic/versions/726cb0f70c6d_link_srpmbuildmodel_to_jobtriggermodel.py

```diff
@@ -0,0 +1,34 @@
+"""link srpmbuildmodel to jobtriggermodel
+
+Revision ID: 726cb0f70c6d
+Revises: adbdc1c21d7e
+Create Date: 2020-08-05 18:14:20.277673
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "726cb0f70c6d"
+down_revision = "adbdc1c21d7e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "srpm_builds", sa.Column("job_trigger_id", sa.Integer(), nullable=True)
```

Comment:
Could some of the current values be populated with the corresponding `job_trigger_id` from the `copr_builds` table?

Reply:
Yes, I can make a script to copy those over but someone will have to run it manually

Reply:
Wait, cannot this be done as part of the migration? I'm no expert in this, just asking :thinking:

Reply:
I dont know. This migration was autogenerated but can be manually tweaked, not sure if were allowed to do any random python code here.

I was thinking of something like:
```
builds = CoprBuildModel.get_merged_chroots()
for build in builds:
    project_details = build.project
    srpm_build = build.srpm_build
    <create a trigger, but now that i think of it, i have no idea how to do it this way either>

```

Reply:
@lbarcziova any advice how to do this? IIRC you did something like this in the past.

Reply:
Do you mean something like [this](https://github.com/packit/packit-service/blob/362302f08d54a0de3fee028ae6a01881569bb5f6/alembic/versions/c6250555a36c_.py#L282) or [this] (https://github.com/packit/packit-service/blob/362302f08d54a0de3fee028ae6a01881569bb5f6/alembic/versions/d90948124e46_add_tables_for_triggers_koji_and_tests.py#L277)?

Reply:
Yes, that's what we are looking for. Do you have some place in the docs we could learn more about this? I wasn't able to find anything.

Reply:
When I was implementing it, these were helpful to read: 
https://stackoverflow.com/questions/24612395/how-do-i-execute-inserts-and-updates-in-an-alembic-upgrade-script
http://ominian.com/2019/07/11/data-migration-with-sqlalchemy-and-alembic/

Reply:
I agree here with Laura, you can easily do that in the migration script like we do in those two files. Just don't forget to define models also in the migration script so we preserve the state of the model which corresponds to the state the migration is done. (Those models can have only a subset of attributes -- only those you need.)

Reply:
Thanks @lbarcziova @lachmanfrantisek